plutus-metatheory/src/Untyped/CEK.lagda.md

```diff
@@ -118,242 +122,290 @@ V-I : ∀ b
 V-I b {tm = zero}   bt = V-I⇒ b bt
 V-I b {tm = suc tm} bt = V-IΠ b bt
 
+{-
+The BUILTIN function provides the semantics of builtin functions.
+
+WARNING: This untyped BUILTIN function implements all builtin functions, but not all of them have been tested.
+This WARNING will be removed once the tests are done.
+-}
+
+{-
+getDATA :  List TermCon → Either RuntimeError (List DATA)
+getDATA [] = return []
+getDATA (pdata x ∷ xs) = do
+     ds ← getDATA xs
+     return (x ∷ ds) 
+getDATA (_ ∷ xs) = inj₁ userError
```

Comment:
We should use some kind of prelude in the formalization, rolling out your own inlined version of `traverse` at multiple call sites must be really annoying.

Reply:
That shouldn't be there. Oh, it is commented. I'll remove the comment.